### ISSUE TITLE
__proto__ can be duplicated in some cases

### ISSUE THREAD

**Issue by ariya** [eslint/espree#176]
Test case:

```js
require('espree').parse('({ __proto__: null, set __proto__(x){} })', { ecmaFeatures: { objectLiteralDuplicateProperties: true } })
```

Actual: Error: Line 1: Object literal may not have data and accessor property with the same name

Expected: a valid syntax tree.

**Comment by nzakas** [comment]
Thanks!

**Comment by nzakas** [comment]
Fixed in 3.0.0.
